common/app/controllers/DevComponentController.scala

```diff
@@ -17,4 +18,8 @@ class DevComponentController(
     renderer.render(TestComponent).map(Ok(_).withHeaders("Content-Type" -> "text/html"))
   }
 
+  def props(): Action[AnyContent] = Action { request =>
```

Comment:
Is the ultimate goal to request a specific component `props`?

Reply:
Yeah. One idea would be to pass a component ID to the `dev/props` endpoint (e.g. `dev/props/404`) and the `DevComponentController` would find the relevant props for that component and concatenate them to the default props. 

Would be useful to have a place where component-specific props are defined... [`archive/app/ui`](https://github.com/guardian/frontend/tree/master/archive/app/ui) is starting to look like that place.